pyproject.toml

```diff
@@ -39,8 +39,12 @@ dependencies = [
     "matplotlib>=3.4"
 ]
 
```

Comment:
readme and docs page should explain how somebody can pip install CleanVision with all optional dependencies, or with none of them

Reply:
Updated